README.md

```diff
@@ -100,6 +100,7 @@ You can find a description of what these parameters are used for [here](https://
 The following additional arguments are supported for the `fluentd` shim logger binary, which can be used to send container logs to  [Fluentd](https://www.fluentd.org). Note that all of these are optional arguments.
 * fluentd-address: The address of the Fluentd server to connect to. By default, the `localhost:24224` address is used.
 * fluentd-async-connect: Specifies if the logger connects to Fluentd in background. Defaults to `false`.
+* fluentd-sub-second-precision: This tag is set to `true` to generate logs in nanosecond resolution.
```

Comment:
nit: Can we add "Defaults to `true`" to this line, so we know that even if a customer doesn't specify this parameter, it'll default to true?

Reply:
By default, fluentd sets this flag to `false` https://docs.docker.com/config/containers/logging/fluentd/#fluentd-sub-second-precision 

We want this flag to be enabled (set to true) for all our customers so that they can receive logs in nanosecond resolution. This is the reason why I did not want to word it as "default to true".  

Please let me know if there is a better way to word this with more clarity.

Reply:
If that's the case, I think it's better to set the default value to false to be consistent with docker's doc.

Reply:
Sorry, I'm still not super clear here. Since fluentd defaults to `false` for this flag, is it safe and product-approved for us to be setting this flag to `true` by default for all fluentd customers?

Reply:
FireLens uses fluentd log driver under the hood, which by default does not send time stamp with nano second precision. The goal is to set this flag to true always when FireLens is used.  This change has been approved by product.

As per my understanding, shim logger is only used in fargate and fargate relies on firelens not the fluentd log driver. So this change should affect only firelens.

You can refer this for more details https://github.com/aws/amazon-ecs-agent/pull/2538#issuecomment-662162363

Reply:
Gotcha, thanks for the clarification. I was getting Fargate's logging capabilities mixed up with ECS on EC2.

Personally, I agree with @xjhe here. I don't mind us setting the default value to `true` here, but would really appreciate a blurb saying "Defaults to `true`" along with a note regarding this deviation between default Docker behaviour.

Reply:
I agree, we should add a note about the deviation from default behavior. I have updated the README.